include/quicly/loss.h

```diff
@@ -181,9 +182,11 @@ inline void quicly_rtt_update(quicly_rtt_t *rtt, uint32_t latest_rtt, uint32_t a
     assert(rtt->smoothed != 0);
 }
 
-inline uint32_t quicly_rtt_get_pto(quicly_rtt_t *rtt, uint32_t max_ack_delay, uint32_t min_pto)
+inline uint32_t quicly_rtt_get_pto(quicly_rtt_t *rtt, const quicly_loss_conf_t *conf, uint32_t max_ack_delay)
 {
-    return rtt->smoothed + (rtt->variance != 0 ? rtt->variance * 4 : min_pto) + max_ack_delay;
+    if (rtt->smoothed == 0)
+        return 2 * conf->default_initial_rtt;
```

Comment:
While this special-casing is fine by itself, I would like to point out that this path is not going to be used to determine when to retransmit, when the endpoint has not obtained an RTT sample.

This is because the spec (and our code) has a special rule in the SetLossDetectionTimer function that says:
```python
  // Use a default timeout if there are no RTT measurements
  if (smoothed_rtt == 0):
    timeout = 2 * kInitialRtt
```

An endpoint sends out packets at a constant speed (every once per 2 * kInitialRtt), until it receives a packet from the peer (servers are also subject to the 3x amplification limit). That is my understanding of the spec, and that is what quicly does.

Now, I have two questions:
* Is this how endpoints should behave? I might argue that endpoints should do an exponential back-off.
* (Assuming that we would retain a separate rule in SetLossDetectionTimer) does the calculation match that we have in the PTO definition? Shouldn't the calculation be `3 * kInitialRtt`?

Reply:
> * Is this how endpoints should behave? I might argue that endpoints should do an exponential back-off.

It turns out that this is an error in quicly, not in spec.

> (Assuming that we would retain a separate rule in SetLossDetectionTimer) does the calculation match that we have in the PTO definition? Shouldn't the calculation be 3 * kInitialRtt?

Filed a spec issue.